app/assets/javascripts/taxon_changes/show.js

```diff
@@ -60,4 +60,32 @@ $( document ).ready( function () {
     $( ".spinner" ).hide();
   } );
 
+  var potential_clashes_link = $( ".potential_clashes_link" )
+  var old_ancestors = potential_clashes_link.data( "input-ancestry" )
+  if( $.isNumeric( old_ancestors ) ){
+    old_ancestors = [ String( old_ancestors ) ];
+  }else{
```

Comment:
I guess we don't have instructions for setting up eslint, but for my sanity please put spaces around `else`.

Reply:
fixed